sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -37,7 +37,10 @@ export type AuthorizationRule = {
 // @public
 export interface CorrelationRuleFilter {
     applicationProperties?: {
-        [key: string]: string | number | boolean | Date;
+        [key: string]: string | number | boolean | Date | {
```

Comment:
I wonder if we should name the new type so it looks a little cleaner (and to make it potentially usable elsewhere), e.g.

```ts
interface Duration {
  val: string;
  kind: "duration";
}
```

but maybe that's formalizing it too much?

Reply:
I was afraid that we'd be reserving the "Duration" word.

Reply:
How about SBDuration?

Reply:
I'm good with that name. /cc @bterlson @ramya-rao-a in case we're painting the bikeshed the wrong color

Reply:
`Iso8601Duration`? are there other formats of durations that might be supported in the future? maybe `kind` can be more specific too `kind: "iso8601"`?

Reply:
I also like iso8601Duration for what it's worth.

Reply:
Good Idea!! Thanks, @jeremymeng!

Going with `Iso8601Duration`.

Reply:
Updated to 
```
ISO8601Duration {
value: string,
kind: "iso8601"
}
```